spec/presenters/summary/components/property_spec.rb

```diff
@@ -0,0 +1,276 @@
+                    complete?: true,
+                    property_owners: [property_owner],
+                    include_partner?: client_has_partner,
+                    crime_application: crime_application, **attributes)
+  }
+
+  let(:property_owner) {
+    instance_double(PropertyOwner,
+                    name: 'Joe',
+                    relationship: relationship,
+                    custom_relationship: 'xyz',
+                    percentage_owned: 10)
+  }
+  let(:crime_application) { instance_double(CrimeApplication, id: 'APP123') }
+  let(:client_has_partner) { false }
+  let(:relationship) { 'friends' }
+
+  let(:attributes) do
+    {
+      id: 'PROPERTY123',
+      crime_application_id: 'APP123',
+      property_type: 'residential',
+      house_type: 'custom',
+      custom_house_type: 'custom_house_type',
+      size_in_acres: nil,
+      usage: nil,
+      bedrooms: 3,
+      value: 200_000,
+      outstanding_mortgage: 100_000,
+      percentage_applicant_owned: 70,
+      percentage_partner_owned: 50,
+      is_home_address: is_home_address,
+      has_other_owners: has_other_owners,
+      address: { city: 'London', postcode: 'TW7' },
+    }
+  end
+
+  let(:is_home_address) { 'yes' }
+  let(:has_other_owners) { 'yes' }
+
+  before { component }
+
+  describe 'actions' do
+    describe 'change link' do
+      it 'show the correct change link' do
+        expect(page).to have_link(
+          'Change',
+          href: '/applications/APP123/steps/capital/residential_property/PROPERTY123',
+          exact_text: 'Change Residential property'
+        )
+      end
+    end
+
+    describe 'remove link' do
+      it 'show the correct remove link' do
+        expect(page).to have_link(
+          'Remove',
+          href: '/applications/APP123/steps/capital/properties/PROPERTY123/confirm_destroy',
+          exact_text: 'Remove Residential property'
+        )
+      end
+    end
+  end
+
+  describe 'answers' do
+    it 'renders as summary list' do # rubocop:disable RSpec/ExampleLength
+      expect(page).to have_summary_row(
+        'Which type of property is it?',
+        'custom_house_type'
+      )
+      expect(page).to have_summary_row(
+        'How many bedrooms are there?',
+        '3',
+      )
+      expect(page).to have_summary_row(
+        'How much is the property worth?',
+        '£200,000.00',
+      )
+      expect(page).to have_summary_row(
+        'How much is left to pay on the mortgage?',
+        '£100,000.00',
+      )
+      expect(page).to have_summary_row(
+        'What percentage of the property does your client own?',
+        '70%',
+      )
+      expect(page).to have_summary_row(
+        'Is the address of the property the same as your client’s home address?',
+        'Yes',
+      )
+      expect(page).to have_summary_row(
+        'Does anyone else own part of the property?',
+        'Yes',
+      )
+    end
+
+    context 'when client has no home address' do
```

Comment:
The wording of this just made me realise something!! 

If the client has no home address recorded in their client details, then the radio:
`Is this the same address as the client's home address?`
doesn't show, and the form blows up when you try to submit it. I only just realised that's why.